pylint_django/checkers/forms.py

```diff
@@ -44,6 +44,11 @@ def visit_classdef(self, node):
             if not isinstance(child, Assign):
                 continue
 
-            if child.targets[0].name == 'exclude':
-                self.add_message('W%s04' % BASE_ID, node=child)
-                break
+            # Capture and ignore AttributeError raised when targets[0] does not
+            # have an attribute "name"
+            try:
+                if child.targets[0].name == 'exclude':
```

Comment:
a better solution might be to guard this with `isinstance(child.targets[0], WhateverExpectedType) and ...`

Reply:
I completely agree, but wouldn't this incur in a longer execution time?

Reply:
imo it's better to have that and not hide bugs (`except AttributeError` could hide any number of other things here) -- this could also be added to the `if isinstance(...): continue` at the top possibly

Reply:
Actually that line filters the children and the problem is in the `targets` items. But you're right about hiding other problems. I'm on it!